.github/workflows/publish.yml

```diff
@@ -0,0 +1,27 @@
+name: Publish
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - name: Set up Python 3.9
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.9'
+      - name: Install Poetry
+        uses: abatilo/actions-poetry@v2.1.4
+        with:
+          poetry-version: 1.1.12
+      - name: Publish
+        run: |
+          poetry version ${GITHUB_REF##*/v}
+          poetry publish --build
+        env:
+          POETRY_PYPI_TOKEN_PYPI: ${{ secrets.PYPI_TOKEN }}
```

Comment:
I haven't tried this way to publish to pypi but if it causes issues, you can use [this existing action ](https://github.com/JRubics/poetry-publish)

Reply:
I have used this before, should work great as soon as we set up the tokens for it in the repo!